### ISSUE TITLE
log scale for any type of chart

### ISSUE THREAD

**Issue by lemairecarl** [comet-ml/issue-tracking#325]
Logarithmic scaling should be available everywhere, since we often deal with ranges that cover multiple orders of magnitude. For instance, if I'm doing a hyperparameter search, and I'm trying values (0.001, 0.01, 0.1, 1), the first two values will probably appear on top of each other on an axis because the linear scale is not sensitive to magnitude.
Thanks.

**Comment by dsblank** [comment]
@lemairecarl Yes, good point. This is on our list, and we hope to have a solution available soon. Thanks!

**Comment by mrvadims** [comment]
We have updated the behavior of the "log scale" transformation to perform logarithmic scaling, instead of applying the log function to the data itself. The old behavior is still available under "log(value)" transformation. The latest version is up in the Production environment.

![image](https://user-images.githubusercontent.com/2479256/76347014-4baa4a80-62dc-11ea-9779-dde9a75c60f3.png)

**Comment by lemairecarl** [comment]
I am not concerned about the line charts. It's really about the "parallel coordinates" chart that I'm talking about.

For instance, look at the "weight decay" parameter below:

![Screenshot from 2020-09-30 10-55-32](https://user-images.githubusercontent.com/13444373/94702242-87097680-030b-11eb-92c4-405d3189be03.png)

The range of the search for weight decay is 1e-8 to 1e-5. For this parameter, all axis labels show zero, and values near the bottom of the range are all crammed together. A log scale would be very useful.

You offer the "log uniform" sampling in your meta optimization engine. For a parameter sampled this way, the parallel coordinates chart will never be really useful, until log scaling is available.

Thank you.
